pkg/kubelet/dockertools/manager.go

```diff
@@ -435,20 +454,6 @@ func (dm *DockerManager) GetPodStatus(pod *api.Pod) (*api.PodStatus, error) {
 	podFullName := kubecontainer.GetPodFullName(pod)
 	uid := pod.UID
```

Comment:
Suggestion: add a comment here stating the limitation of this function. E.g., if all dead containers were garbage collected, the container status may be inaccurate (including state and restart count).